File: Python Code/Daily_Flash_Code_0/Week 2/22Jan/08-DailyFlash_Solutions/08-DailyFlash_Solutions/22_Jan_Solutions_Three/Python/p1.py
'''
Program 1: Write a program that accepts two integers from user and prints
addition & Subtraction of them.
{Note: checks for greater number to subtracts with while subtracting numbers}
Input: 10 20
Output:
Addition is 20
Subtraction is 10
'''






var1=int(input("enter first integer "))
var2=int(input("enter second integer "))

add=var1+var2
print("Addition is ",add)

if(var1 > var2):
	sub=var1-var2
	print("Subtraction is ",sub)
elif(var2>var1):
	sub=var2-var1
	print("Subtraction is ",sub)

File: Python Code/Daily_Flash_Code_0/Week 2/22Jan/08-DailyFlash_Solutions/08-DailyFlash_Solutions/22_Jan_Solutions_Three/C++/p1.cpp
/*
Program 1: Write a program that accepts two integers from user and prints
addition & Subtraction of them.
{Note: checks for greater number to subtracts with while subtracting numbers}
Input: 10 20
Output:
Addition is 20
Subtraction is 10
*/


#include<iostream>


int main(){
	int var1,var2,sub=0,add=0;

	std::cout<<"Enter two integers : ";
	std::cin>>var1;
	std::cin>>var2;

	add=var1+var2;
	std::cout<<"addition is "<<add<<"\n";


	if(var1>var2){
		sub=var1-var2;
		std::cout<<"Subtraction is "<<sub<<"\n";

	}
	else if(var2>var1){
		sub=var2-var1;
		std::cout<<"Subtraction is "<<sub<<"\n";

	}




	return 0;
}
